docs/book/src/user/quick-start.md

```diff
@@ -175,6 +175,27 @@ and `kubeadm` control-plane providers.
 Depending on the infrastructure provider you are planning to use, some additional prerequisites should be satisfied
 before getting started with Cluster API. See below for the expected settings for common providers.
 
+The general steps to setup the specific infrastructure provider is given below, these steps can be followed for all 
```

Comment:
As far as I'm aware CAPA is the only provider which has an additional binary and it is documented in the AWS tab (right below)

Reply:
(/cc @randomvariable)

Reply:
In case the steps for AWS we already have are not clear enough, do we want to integrate this new doc in the AWS tab?

I'm just a bit hesitant to add this to the general part of the doc when AWS is the only provider which has an additional binary and we already have some doc for the binary in the AWS tab.

Reply:
okay sure I can do that , I will move it under the AWS tab